### PR TITLE
#34963 Breadcrumbs styling update to fit a new style-guide.

### PR DESCRIPTION
**Pull request by kruchy8**
https://github.com/SnowdogApps/magento2-alpaca-components/issues/83

### CODE REVIEW

File: components/02-elements/breadcrumbs/_breadcrumbs.scss [hunk 1]
```diff
@@ -1,33 +1,56 @@
...
-$breadcrumbs-color-second   : $gray;
-$breadcrumbs-clickable-area : 48px;
+$breadcrumbs-margin         : 8px !default;
+$breadcrumbs-padding        : 8px !default;
+$breadcrumbs-clickable-area : 48px !default;
+$breadcrumbs-color          : $gray-dark !default;
+$breadcrumbs-font-family    : $font-family-base !default;
+$breadcrumbs-font-size      : $font-size-small !default;
+$breadcrumbs-line-height    : $font-line-height !default;
+//Active
```
Comment (anqaka) [inline components/02-elements/breadcrumbs/_breadcrumbs.scss]: 1. IMO comment is not necessary, we now that is active, it's from code.
2. use BEM notification for variables `$breadcrumbs__padding` etc

File: components/02-elements/breadcrumbs/_breadcrumbs.scss [hunk 2]
```diff
@@ -1,33 +1,56 @@
...
+$breadcrumbs-color-active   : $color-primary !default;
+$breadcrumbs-font-family-active: $font-family-base-bold !default;
```
Comment (anqaka) [inline components/02-elements/breadcrumbs/_breadcrumbs.scss]: align variables

File: components/02-elements/breadcrumbs/breadcrumbs.hbs [hunk 3]
```diff
@@ -5,9 +5,11 @@
                 {{ this }}
             </span>
         {{else}}
-            <a href="#" class="breadcrumbs__item">
-                {{ this }}
-            </a>
+            <span class="breadcrumbs__item">
+                <a href="#" class="breadcrumbs__link">
```
Comment (anqaka) [inline components/02-elements/breadcrumbs/breadcrumbs.hbs]: set a href as a context element in config, please
add also attribute `aria-label` with text setting in config file (in context)

File: components/02-elements/breadcrumbs/_breadcrumbs.scss [hunk 4]
```diff
@@ -1,33 +1,56 @@
-$breadcrumbs-padding        : 10px;
-$breadcrumbs-margin         : 10px;
-$breadcrumbs-color          : $font-color-base; 
-$breadcrumbs-color-second   : $gray;
-$breadcrumbs-clickable-area : 48px;
+$breadcrumbs-margin         : 8px !default;
+$breadcrumbs-padding        : 8px !default;
+$breadcrumbs-clickable-area : 48px !default;
```
Comment (anqaka) [inline components/02-elements/breadcrumbs/_breadcrumbs.scss]: I get the deal, that should be 48px height... but... when width of the screen is smaller and the breadcrumbs are in two lines, the space between the line is too large. What do you think @kruchy8 @Igloczek

Comment (kruchy8) [reply components/02-elements/breadcrumbs/_breadcrumbs.scss]: I have consulted it with Michal and the decision was to leave it with 48px.

File: components/02-elements/breadcrumbs/_breadcrumbs.scss (hunk 2 above)
Comment (kruchy8) [reply components/02-elements/breadcrumbs/_breadcrumbs.scss]: Done

File: components/02-elements/breadcrumbs/_breadcrumbs.scss (hunk 1 above)
Comment (kruchy8) [reply components/02-elements/breadcrumbs/_breadcrumbs.scss]: Done

File: components/02-elements/breadcrumbs/breadcrumbs.hbs (hunk 3 above)
Comment (kruchy8) [reply components/02-elements/breadcrumbs/breadcrumbs.hbs]: Done

File: components/02-elements/breadcrumbs/_breadcrumbs.scss [hunk 5]
```diff
@@ -41,12 +40,12 @@ $breadcrumbs-font-family-active: $font-family-base-bold !default;
...
+        padding: 0 $breadcrumbs__padding;
+        font-family: $breadcrumbs__font-family;
+        color: $breadcrumbs__color;
 
         &:hover {
-            color: $breadcrumbs-color-active;
+            color: $breadcrumbs__color--active;
         }
         &:active,
         &:focus {
```
Comment (anqaka) [inline components/02-elements/breadcrumbs/_breadcrumbs.scss]: line 55 - if you use `outline: none`, please put it in variable using global variable, b'c we have to change it

**Comment by anqaka** [comment]
I added tabindex on the last span element to make it focusable (span element isn't focusable by default) and  update the aria-label text

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [Unreleased]
+### Added
+- Breadcrumbs
```
Comment (anqaka) [inline CHANGELOG.md]: should we add here information that you've added a global variable `$outline-base`? IMO we should

Comment (kruchy8) [reply CHANGELOG.md]: According to the link from alpaca rocket channel (http://keepachangelog.com/en/1.0.0) :
```
The purpose of a commit is to document a step in the evolution of the source code. Some projects clean up commits, some don't.

The purpose of a changelog entry is to document the noteworthy difference, often across multiple commits, to communicate them clearly to end users.
```
I guess we don't have to put this kind of info within this file. I have updated this message, check it out.

Comment (anqaka) [reply CHANGELOG.md]: OK, it's clear inside the commit. lets leave it

**Pull request merged**